bibliography.md

```diff
@@ -0,0 +1,14 @@
+
+Toward a readable IPFS Bibliography:
+====================================
+
+   * [Kademlia: A Peer-to-peer Information System Based on the XOR Metric](http://pdos.csail.mit.edu/~petar/papers/maymounkov-kademlia-lncs.pdf) by Petar Maymounkov and David Mazieres
+   * [S/Kademlia: A Practicable Approach Towards Secure Key-Based Routing](http://www.researchgate.net/publication/4319659_SKademlia_A_practicable_approach_towards_secure_key-based_routing/links/02e7e524ad3e97d67d000000) by Ingmar Baumgart and Sebastian Mies
+   * [Escaping the Evils of Centralized Control with self-certifying pathnames](http://www.scs.stanford.edu/~dm/home/papers/mazieres:escape.ps.gz)
```

Comment:
Maybe link http://www.sigops.org/ew-history/1998/papers/mazieres.ps instead, and add in `by David Mazieres`